File: solutions/lua/raindrops/1/raindrops.lua
return function(n)

  local res = ""
  local check = {false, false, false}
  if n % 3 == 0 then
    res = (res .. "Pling")
    check[1] = true
  end
  if n % 5 == 0 then
    res = (res .. "Plang")
    check[2] = true
  end
  if n % 7 == 0 then
    res = (res .. "Plong")
    check[3] = true
  end

  if check[1] == false and check[2] == false and check[3] == false then
    res = tostring(n)
  end

  return res
  
end

File: solutions/cpp/raindrops/1/raindrops.cpp
#include "raindrops.h"

namespace raindrops {

// TODO: add your solution here
std::string convert(int n) {
    std::string res;
    
    if(n % 3 == 0) { res.append("Pling"); }
    if(n % 5 == 0) { res.append("Plang"); }
    if(n % 7 == 0) { res.append("Plong"); }

    if(n % 3 != 0 && n % 5 != 0 && n % 7 != 0) {
        res.append(std::to_string(n)); }

    return res;
}

}  // namespace raindrops
